app/upgrades/v10/upgrades.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/module"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 
-	"github.com/cosmos/gaia/v10/app/keepers"
+	"github.com/cosmos/gaia/v11/app/keepers"
```

Comment:
Why is this in `app/upgrades/v10` folder?

Reply:
so v10 app keepers is called, v10 keeper is the same keeper as v11. for the upgrade folder v7-v9, they are also calling v11 keeper. the issues is they are legacy, do we importing gaia v7-v10 , for  just using the v7 - v10 keepers which are not used in v11. the legacy code should be deleted maybe.  @mpoke

Reply:
actually if i import gaia/v9 into v11, there is err when compile.

Reply:
> actually if i import gaia/v9 into v11, there is err when compile.

Yeah that's supposed to happen.

What I'm saying is, we should only keep the latest upgrade in `app/upgrades`